src/internal/deployment.js

```diff
@@ -23,6 +23,8 @@ const finalErrorStatus = {
   deployment_lost: 'Deployment failed to report final status.'
 }
 
+const maxTimeout = 600000
```

Comment:
💡 **Idea**

Could consider uppercasing and snake-casing this name since it is a constant, but that's definitely not a hard rule in JavaScript practices. I'm fine leaving it as-is, just sharing some thoughts. 🤷🏻 

```suggestion
const MAX_TIMEOUT = 600000
```

:warning: If so, we'll need to update all of its references as well.